tempesta_fw/http_parser.c

```diff
@@ -1237,6 +1207,63 @@ __req_parse_x_forwarded_for(TfwHttpMsg *msg, unsigned char *data, size_t len)
 	return r;
 }
 
+/* In request we should pass empty headers:
+ * RFC 7230 5.4:
+ * ....
+ * ....
+ * If the authority component is missing or
+ * undefined for the target URI, then a client MUST send a Host header
+ * field with an empty field-value.
+ *
+ * ATTENTION: using of RGEN_LWS_empty should be corresponding with
+ * tfw_http_msg_hdr_val function
+ */
+#define RGEN_LWS_common_cases(move_state)			\
```

Comment:
Please use the name conventions that are already used in the macros. In this case, I believe, it should be `(st)`.